daemon/algod/api/server/v2/handlers.go

```diff
@@ -966,7 +966,9 @@ func (v2 *Handlers) TransactionParams(ctx echo.Context) error {
 	return ctx.JSON(http.StatusOK, response)
 }
 
-type preEncodedTxInfo struct {
+// PreEncodedTxInfo represents the PendingTransaction response before it is
+// encoded to a format.
+type PreEncodedTxInfo struct {
```

Comment:
Made this struct public so we can parse `PendingTransactionInformation` responses in the REST client.